src/SSHLibrary/abstractclient.py

```diff
@@ -104,6 +105,9 @@ def shell(self):
             self._shell = self._create_shell()
         return self._shell
 
+    def banner(self):
+        return self.banner
```

Comment:
- This probably should be `get_banner`.
- Should just `raise NotImplementedError`.
- Should be moved next to the other banner getting method.